example/package.json

```diff
@@ -23,7 +23,7 @@
     "react-native": "0.63.4",
     "react-native-animated-pagination-dots": "^0.1.51",
     "react-native-gesture-handler": "^1.9.0",
-    "react-native-reanimated": "^1.13.2",
+    "react-native-reanimated": "^2.3.1",
```

Comment:
I don't think reanimated was being used in the example so this should not cause any issues!